kivy/core/audio/audio_sdl2.pyx

```diff
@@ -42,7 +43,7 @@ cdef mix_init():
         mix_is_init = -1
         return 0
 
-    mix_flags = Mix_Init(0)
+    mix_flags = Mix_Init(MIX_INIT_FLAC|MIX_INIT_MOD|MIX_INIT_MP3|MIX_INIT_OGG)
```

Comment:
There is an issue with FLAC files - mix_flags gets MIX_INIT_FLAC (and the library is installed), but Mix_LoadWAV() returns NULL. Maybe it has to do with my SDL version

MIX_INIT_FLAC could just be removed from Mix_Init(). At least tests will pass on a system with only sdl2 audio provider, as long as libvorbis is available.